src/lib/zodSchema.ts

```diff
@@ -0,0 +1,9 @@
+import { z } from 'zod';
+
+export const TeamSchema = z.object({
+  name: z.string(),
+  id: z.string(),
```

Comment:
Can you please confirm if this works?
```suggestion
  id: z.string().uuid(),
```
I think yes, and that way you are more strict on your validation

Reply:
yes it does :)